.travis.yml

```diff
@@ -41,12 +22,31 @@ jobs:
           script: py.test --cov=scout -rxs tests/
           after_success: coveralls
 
-        - name: "Code formatting"
+        - name: "Package Install"
+          if: type = pull_request
+          install: python setup.py install
+          script: pip check
+
+        - name: "Production Dependencies"
+          if: type = pull_request
+          install:
+              - pip install cython
+              - pip install .
+          script: pip check
+
+        - name: "Development Dependencies"
+          if: type = pull_request
+          install:
+              - pip install cython
+              - pip install . -r requirements-dev.txt
+          script: pip check
+
+        - name: "PEP8 Compliance"
           if: type = pull_request
           install: pip install black
           script: git --no-pager diff --name-only --diff-filter=AM $TRAVIS_COMMIT_RANGE | grep -F ".py" | xargs black --check
 
-        - name: "Pylint score"
+        - name: "Code Quality Score"
```

Comment:
Hm, I kind of liked pylint score better. Judging from the lints its a lot about whitespace around commas and assignment operators. Its nice and tidy to have that look the same everywhere, but does not affect function in any way.

Reply:
ok, changed to Pylint Rating

Reply:
We'll change it later if needed..